olp-cpp-sdk-authentication/src/AuthenticationClient.cpp

```diff
@@ -530,7 +531,7 @@ client::CancellationToken AuthenticationClient::Impl::SignInClient(
   };
 
   context.ExecuteOrCancelled(
-      [&]() { return GetTimeFromServer(std::move(time_callback)); }, []() {});
+      [&]() { return GetTime(std::move(time_callback)); }, []() {});
```

Comment:
```suggestion
      [&]() { return GetTime(std::move(time_callback)); });
```
Cancellation function can be ommited.